### ISSUE TITLE
can't run account

### ISSUE THREAD

**Issue by colapig** [nebulasio/neb.py#2]
python3 mac os
 from Crypto.Hash import SHA3_256 as SHA3
ImportError: cannot import name 'SHA3_256'

**Comment by evanrang** [comment]
https://github.com/nebulasio/neb.py/pull/3

**Comment by BrushXiaoMinGuo** [comment]
@colapig  hi, colapig. please you can try this:
sudo pip uninstall crypto
sudo pip uninstall pycrypto
sudo pip install pycrypto

**Comment by colapig** [comment]
@BrushXiaoMinGuo yes I have tried this but not succeed for me... finally I fix the account by doing
 sudo pip3 install crypto==1.4.1
I don't know Y but it's work well now.
